examples/forward/plot_frequency_interference.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+"""
+.. _ex-frequency-interference:
+
+Frequency Interference
+======================
+
+This example shows how the spatial offset of nearly phase-locked source time
+courses with lower frequency oscillations have much less destructive
+interference than those with higher frequencies leading to a 1/f power
+spectral density (psd) distribution. This phenomena is commonly
+misattributed to the skull acting as a lowpass filter.
+
+.. contents:: This tutorial covers:
+   :local:
+   :depth: 2
+
+"""
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#          John Mosher   <John.C.Mosher@uth.tmc.edu>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+from mne.minimum_norm import read_inverse_operator, apply_inverse
+from mne.time_frequency import psd_array_multitaper
+from mne.simulation import simulate_evoked
+
+
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+sample_dir = op.join(data_path, 'MEG', 'sample')
+
+# Read evoked data
+fname_evoked = op.join(sample_dir, 'sample_audvis-ave.fif')
+evoked = mne.read_evokeds(fname_evoked, condition=0, baseline=(None, 0))
+
+
+##############################################################################
+# Visualize 1/f power spectral density in m/eeg data
+# --------------------------------------------------
+#
+# The power decreases with increasing frequency, this is usually an
+# exponential decay.
+
+psds, freqs_psd = psd_array_multitaper(
+    evoked.data, evoked.info['sfreq'], fmin=2, fmax=50)
+fig, axes = plt.subplots(1, 3, figsize=(10, 5))
+axes[0].plot(freqs_psd, psds[mne.pick_types(evoked.info, meg='mag')].T)
+axes[0].set_title('mag')
+axes[1].plot(freqs_psd, psds[mne.pick_types(evoked.info, meg='grad')].T)
+axes[1].set_title('grad')
+axes[2].plot(freqs_psd, psds[mne.pick_types(evoked.info, eeg=True)].T)
+axes[2].set_title('eeg')
+fig.suptitle('1/f Power Decay in Real Evoked')
+axes[0].set_ylabel('Power')
+axes[1].set_xlabel('Frequency (Hz)')
+fig.show()
```

Comment:
usually to show the 1/f you plot in loglog as in:

![image](https://user-images.githubusercontent.com/161052/102590052-44885680-4110-11eb-9f26-d9c35cce32b3.png)